proseco/core.py

```diff
@@ -996,6 +1003,28 @@ def null_logger(*args, **kwargs):
 
             return null_logger
 
+    def set_agasc_supp_version(self):
+        """
+        Get the versions of the agasc supplement used for the star data for catalog
+        determination and save in the appropriate MetaAttributes.
+        """
+
+        # It seems incorrect to save the supplement versions that were used to
+        # update stars if the supplement was not actually enabled/used.
+        # This block copies the logic from agasc.update_from_supplement
+        # (which perhaps should be generalized in the interface).
+        supplement_enabled_env = os.environ.get(agasc.SUPPLEMENT_ENABLED_ENV,
+                                                agasc.agasc.SUPPLEMENT_ENABLED_DEFAULT)
+        if supplement_enabled_env not in ('True', 'False'):
+            raise ValueError(
+                f'{agasc.SUPPLEMENT_ENABLED_ENV} env var must be either "True" or "False" '
+                f'got {supplement_enabled_env}')
+        if supplement_enabled_env == 'True':
+            agasc_versions = get_supplement_table('agasc_versions')
+            self.supp_agasc_version = agasc_versions['supplement'][0]
+            last_updated = get_supplement_table('last_updated')
+            self.supp_last_updated = last_updated['supplement'][0]
```

Comment:
It's worth a comment here that `last_updated` is a 1-row table, and that `supplement` is the column that provides the time of last update of any table of the supplement (besides the version tables themselves).

Reply:
And do we want another convenience method in `agasc` for it or comment it here in `proseco`.